py/nlr.h

```diff
@@ -89,9 +90,16 @@
 #elif defined(__mips__)
     #define MICROPY_NLR_MIPS (1)
     #define MICROPY_NLR_NUM_REGS (MICROPY_NLR_NUM_REGS_MIPS)
-#elif defined(__riscv) && defined(__riscv_xlen) && (__riscv_xlen == 32)
-    #define MICROPY_NLR_RV32I (1)
-    #define MICROPY_NLR_NUM_REGS (MICROPY_NLR_NUM_REGS_RV32I)
+#elif defined(__riscv)
+    #if (__riscv_xlen == 32)
```

Comment:
We usually don't put parenthesis around logic expressions that are on their own, in preprocessor if's.

Reply:
Sorry, this should be fixed now.

Reply:
Thanks!